try{
    let hello= "Rohan";
    console.log(`sucess at every point ${hello}`); 
    let age = 29;
    console.log(age1)
}
catch(err){
    console.log(`occur has occured ${err.message}`);
}
finally{
    console.log("done");
}